src/gazebo_gimbal_controller_plugin.cpp

```diff
@@ -308,9 +308,13 @@ void GimbalControllerPlugin::OnUpdate()
     /// and gimbal is constructed using yaw-roll-pitch-variable-axis
     ignition::math::Vector3d currentAngleYPRVariable(
       this->imuSensor->Orientation().Euler());
+#if GAZEBO_MAJOR_VERSION >= 8
```

Comment:
You don't actually have to wrap this in an #if. This code will also work with the older versions which used ign_math2 and didn't have an explicit tag on the quaternion constructor.